include/solvers/vof.h

```diff
@@ -656,6 +712,10 @@ class VolumeOfFluid
 
   // Conservation Analysis
   TableHandler table_monitoring_vof;
+  double       volume_monitored;
+  double       mass_monitored;
+  double       mass_first_iteration;
+  double       sharpening_threshold;
```

Comment:
It's a very good question! I modified the VOF methods so that it is clear where the sharpening threshold is used, so they can just pass it to each other and we wouldn't need an attribute for that. But, as I wanted to add the sharpening threshold to the output table, I was stuck with the `postprocess` method, which should be defined in the same fashion for all the subphysics (it is called in the `multiphysics_interface`). So I passed the `sharpening_threshold` as an attribute of VOF. This is not very neat, but it seemed to be the easiest route for now.

Reply:
I understand, bit of an hack but that will do.